### DISCUSSION TITLE
Predictable container names and network

### DISCUSSION THREAD

**Discussion by DiiiaZoTe** [coollabsio/coolify#1661] [Feature Requests]
Hi,
When deploying a ressource using GitHub app, either public or private. It would be nice to have a name and network field to set the name and the network of the container. Right now, we have absolutely no visibility on the container. 

Use case:
- Nextjs webapp redeploys on git push, container name is set and predictable.
- Allows us to add another resource, a proxy that redirects certain domains to our app.
Let's say we own 2 domains, example.io and example.com.
Our app is example.io then we could do rewrites or redirects for example.com to example.io 

One could say why don't you containerize the app yourself and just deploy everything using a docker compose... I could but I don't want to have to setup a pipeline to containerize the app every time I push a change.

**Comment by Wulfheart** [comment]
This would be really nice but it also creates an implicit dependence on coolify's behaviour in downstream scripts that might (or might not) use this naming to perform different actions so it would eventually have to be versioned just like API changes.

**Reply by DiiiaZoTe** [reply]
That's a valid point...
